lib/dynflow/rails/configuration.rb

```diff
@@ -32,6 +32,8 @@ class Configuration
       # the orchestration tied to the models.
       attr_accessor :disable_active_record_actions
 
+      attr_accessor :process_role
```

Comment:
I still do not like the name of this config, I feel it missleading. We want to be able to have one sidekiq process for worker and orchestrator together, right?

Reply:
Yeah, it is kinda misleading. If both are in the same process, which is totally doable, they are considered to be orchestrator, but still perform both duties.